app/build.gradle

```diff
@@ -40,5 +40,6 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
 
-    implementation 'com.google.android.material:material:1.1.0'
+    //NavigationView
+    implementation material.material
```

Comment:
Let's do the same for all others dependencies: androidx.core, androidx.appcompat, junit, ...